a3_procurement/doc_events/sales_invoice.py

```
@@ -61,28 +99,28 @@ def patched(doc):
 # ------------------------------------------------------------------------------
 
 def patch_ic_validate_item_gst_details():
-    try:
-        from india_compliance.gst_india.overrides.transaction import ItemGSTDetails
-    except Exception:
-        return
+	try:
+		from india_compliance.gst_india.overrides.transaction import ItemGSTDetails
+	except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)